base/linalg/generic.jl

```diff
@@ -387,9 +387,9 @@ function generic_vecnormp(x, p)
     else
         T = typeof(float(norm(v)))
     end
-    spp::promote_type(Float64, T) = p
+    spp::promote_type(Float64,T) = p
```

Comment:
I see why I did this, the rest of the file had `promote_type` without spaces between